tests/vue/vue/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -52,6 +59,47 @@ semi: false
 =====================================output=====================================
 <template>
   <div
+    v-for="(
+      { longLongProp, longLongProp }, index
+    ) of longLongLongLongLongLongLongLongList"
+    v-for="{
+      longLongProp,
+      longLongProp,
+    } of longLongLongLongLongLongLongLongList"
+    v-for="(
+      {
+        longLongProp,
+        longLongProp,
+        anotherLongLongProp,
+        yetAnotherLongLongProp,
+      },
+      index
+    ) of longLongLongLongLongLongLongLongList"
+    v-for="{
+      longLongProp,
+      longLongProp,
+      anotherLongLongProp,
+      yetAnotherLongLongProp,
+    } of longLongLongLongLongLongLongLongList"
+    v-for="(
+      objectPropKey, { firstValue, secondValue }, index
+    ) in objectWithAVeryLongLongLongLongName"
+    v-for="(
+      objectPropKey, { firstValue, secondValue, thirdValue, fourthValue }, index
+    ) in objectWithAVeryLongLongLongLongName"
```

Comment:
Is this a valid case for vue?

Reply:
Whoopsie, my bad, I accidentally swapped the first and second params.

Reply:
Oh, it is `value, key, index`, never use this syntax 👍